rollup-tests.config.js

```diff
@@ -0,0 +1,39 @@
+import * as glob from 'glob';
```

Comment:
We might want to put the rollup configs in a separate directory if there are lot of them. In this project there are only two however.